### PR TITLE
Bug 1981975: Update service network status to reflect dual stack entries

### PR DESCRIPTION
**Pull request by lmzuccarelli**
This addresses the BZ https://bugzilla.redhat.com/show_bug.cgi?id=1981975

As mentioned in the BZ, if this fix is approved we will need to evaluate to backport the fix

### CODE REVIEW

File: pkg/util/proxyconfig/no_proxy_test.go [hunk 1]
```diff
@@ -176,7 +197,7 @@ func TestMergeUserSystemNoProxy(t *testing.T) {
 			args: args{
 				proxy:   proxyConfigWithNoProxy("172.30.0.1."),
 				infra:   infraConfig(configv1.AWSPlatformType, "^&", "us-west-2"),
-				network: netConfig("10.128.0.0/14", ""),
+				network: netConfig("10.128.0.0/14", []string{""}),
```
Comment (frobware) [inline pkg/util/proxyconfig/no_proxy_test.go:200]: Any reason to explicitly test with `[]string{""}` instead of `nil`?

Comment (lmzuccarelli) [reply pkg/util/proxyconfig/no_proxy_test.go:200]: No reason, I will change it

**Comment by Miciah** [comment]
/retitle Bug 1981975: Update service network status to reflect dual stack entries
Looks good, but please change the base branch from "release-4.9" to "master".  See https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-base-branch-of-a-pull-request for instructions on how to do that.

**Pull request merged**